docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,111 @@
 Latent Tuberculosis
 ===================
 
-.. todo:
+Validation and Verification Criteria
+------------------------------------
 
-   all of this
+Obejective
+++++++++++
+External
+ - Model results should be checked against local TB epidemiology
+    - e.g. rate of decline in burden of disease should be compared
+      with historical evidence.
+ - Compare our results (e.g. ICERs) to similar models or empirical
+   assessments where possible
+Internal
+ - Test hypotheses without develop a full simulation. 
+ - Calibrate simulation baseline against GBD 2017 results
+    - Sim outputs mean should perfectly match GBD results.
+    - Sim outputs uncertainty should be wider than GBD results,
+      because both stochastic and parametric uncertainty are included.
+
+V&V strategy
+++++++++++++
+Model Validation
+ - Check the logical structure and input data for concept model,
+   make sure
+    - the theories and assumptions underlying the conceptual model are correct.
+    - the data to build, evaluate, and test model are correct.
+Model Verification
+ - Check the translation from concept model to script based on test data,
+   make sure
+    - the computer programming and implementation of the concept model is correct.
+    - the output of the model can be calibrated against GBD results.
+
+logic
++++++
+- Parent cause is the sum of child causes
+    - Fatal: Deaths (CSMR, Excess MR), YLLs
+    - Non-fatal: Cases, YLDs, Prevalence, Incidence
+    - DALYs = YLLs + YLDs
+- By location-/age-/sex-
+- Start from count space
+- We expect that total active TB cases less than LTBI cases for sim output
+  of event count
+
+How GBD post-processing results
```

Comment:
I'm a little unclear on the purpose of this section; is this explaining some sort of discrepancy you guys had in your baseline sim vs. gbd? or just to provide context for what you're validating against?

Reply:
These are some tips when we validating against GBD results, I’ll rewrite this section into a readable paragraph!